### DISCUSSION TITLE
"ECONNRESET" when running publish command

### DISCUSSION THREAD

**Discussion by danielapochini** [test-results-reporter/reporter#123] [Q&A] [answered]
Hi there!

When I run the following command:
`npx test-results-reporter publish -c resources/config.json`

I'm getting the error message below:
 
> Error: read ECONNRESET
    at TLSWrap.onStreamRead (node:internal/stream_base_commons:217:20) {
  errno: -4077,
  code: 'ECONNRESET',
  syscall: 'read'
}

config.json:
```
{
   "reports":[
      {
         "targets":[
            {
               "name":"teams",
               "inputs":{
                  "url":"{TeamsWebhook}",
                  "title":"{PROJECT_NAME}",
                  "title_link":"{BUILD_URL}"
               }
            }
         ],
         "results":[
            {
               "type":"junit",
               "files":[
                  "resources/TestResult.xml"
               ]
            }
         ]
      }
   ]
}
```

It was working fine until few days ago.  Can anybody help me? Thx
Version: 1.0.17

**Comment by ASaiAnudeep** [comment, marked as answer]
The issues seems to be unavailability of network connectivity to the MS Teams. Can you make sure `TeamsWebHook` is reachable as expected from the machine where you are trying to publish the results. You can try `curl` command to check.

**Reply by danielapochini** [reply]
It looks like it was a server unavailability indeed, today I ran it and is working fine as expected.
Btw, thanks for the tip about the `curl` command, if it happens again now I know how to investigate it!
